_data/gaming.yml

```
@@ -97,6 +97,13 @@ websites:
     tfa:
       - software
     doc: https://support.faceit.com/hc/en-us/articles/115000254670
+    
+  - name: Fanatical
+    url: https://www.fanatical.com
+    img: fanatical.png
+    tfa:
+      - software
```

```suggestion
      - totp
```
This would be `totp` not just simply `software` from what I see in the doc.